tools/pip-requires

```diff
@@ -19,36 +19,37 @@ pysftp==0.2.9               # forces installation of paramikoi and pycrypto
 pycrypto==2.6.1             # Cryptographic modules
 s3cmd==2.0.1                # Package built from http://s3tools.org/download
 gearman==2.0.2              # Used only gor emulation framework
-stomp.py==4.1.19            # ActiveMQ Messaging Protocol
+stomp.py==4.1.20            # ActiveMQ Messaging Protocol
 dnspython==1.15.0           # To resolve ActiveMQ broker alias
 pystatsd==0.1.10            # Needed to log into graphite with more than 1 Hz
 pygeoip==0.3.2              # GeoIP API
-geoip2==2.6.0               # GeoIP2 API (for IPv6 support)
+geoip2==2.7.0               # GeoIP2 API (for IPv6 support)
 maxminddb==1.3.0            # extension for reading the MaxMind DB format
 threadpool==1.3.2           # threadpool
-cffi==1.11.2                # Foreign Function Interface for Python calling C code
-cryptography==2.1.3         # Cryptographic recipes and primitives
+cffi==1.11.4                # Foreign Function Interface for Python calling C code
+cryptography==2.1.4         # Cryptographic recipes and primitives
 enum34==1.1.6               # Python 3.4 Enum backported
 gcloud==0.18.3              # API Client library for Google Cloud
 googleapis-common-protos==1.5.3 # Common protobufs used in Google APIs
 httplib2==0.10.3            # A comprehensive HTTP client library
-pyOpenSSL==17.4.0           # Python wrapper module around the OpenSSL library
+pyOpenSSL==17.5.0           # Python wrapper module around the OpenSSL library
```

Comment:
Does this mean we can get rid of the hack we are doing at the moment with pyopenssl?

Reply:
I don't know. the hack was for urllib3 no ?

Reply:
Ah yes, you are right
`/python2.7/site-packages/urllib3/contrib/pyopenssl.py`